ui/push-health/MyPushes.jsx

```diff
@@ -30,26 +35,34 @@ class MyPushes extends React.Component {
   constructor(props) {
     super(props);
 
+    this.params = parseQueryParams(this.props.location.search);
     this.state = {
       pushMetrics: [],
       repos: [],
       loading: false,
       selectedRepo: defaultRepo,
+      defaultUser: this.props.user.email || this.params.author,
```

Comment:
No, the`UserModel` calls our `/user/` API whenever someone navigates to any part of Treeherder (each part of the app has its own entry point for the different nested routes) and if there isn't any user data (because someone isn't logged in) an empty object is returned from `UserModel` but we set a default property of `{ isLoggedIn: false }` in `push-health/app`.